README.md

```diff
@@ -139,6 +139,9 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
 - **Electron**
   - Erick Zhao ([@erickzhao](https://github.com/erickzhao), Slack)
   - Michaela Laurencin ([@mlaurencin](https://github.com/mlaurencin), Microsoft)
+- **Express**
+  - Chris de Almeida ([@ctcpip](https://github.com/ctcpip), IBM)
```

Comment:
it's not me, but I don't recall off the top of my head what we decided here

Reply:
removed and force pushed

Reply:
update: should be Ulises and Wes, based on what was agreed at the TC